.github/actions/cache-go-dependencies/action.yaml

```diff
@@ -3,6 +3,7 @@ description: Cache Go Dependencies
 runs:
   using: composite
   steps:
+    - uses: chetan/git-restore-mtime-action@v2
```

Comment:
this is a minor point and i know we don't follow this elsewhere, but it would be more secure to point to a SHA instead of a version, particularly because this is a random guy's code repo.